dashboard/test/factories/reference_guides.rb

```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :reference_guide do
+  end
+end
```

Comment:
per discussion in other PR, I believe this file should not be added in this PR, since my understanding is we aren't planning to use it.